ibm_cloud_sdk_core/authenticators/basic_authenticator.py

```diff
@@ -62,13 +62,13 @@ def validate(self):
                 'Please remove any surrounding {, }, or \" characters.')
 
 
-    def __construct_basic_auth_header(self):
+    def __construct_basic_auth_header(self) -> None:
```

Comment:
I don't think `None` is correct here.  Three lines down you are returning a string.